### ISSUE TITLE
Anterior meningocele

### ISSUE THREAD

**Issue by patt1morgan** [obophenotype/human-phenotype-ontology#7021]
**Preferred term label:**
Anterior meningocele

**Synonyms**

**Definition (free text, please give PubMed ID)** 
Anterior spinal meningoceles  are formed by protrusion of the spinal meninges through defects in the vertebral column.  This disease most commonly results from herniation at the sacral level but can occur higher up, such as cervical and thoracic levels.  PMID: 32243391

**Parent term (use hpo.jax.org/app)** 
Meningocele HP:0002435

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**  PMID: 32243391
NF1 Orph 636
Marfan syndrome Orph 558
Ehlers Danlos Orpha 287

**Your nano-attribution (ORCID)**

**Comment by patt1morgan** [comment]
@jen-campbell, @BimalChaudhari.  Thank you for agreeing to work with us to advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it or help us improve it. Any suggestions welcome!
